cishouseholds/weights/design_weights.py

```diff
@@ -51,7 +52,7 @@ def generate_weights(
         df = assign_tranche_factor(
             df=df,
             column_name_to_assign="tranche_factor",
-            barcode_column="ons_household_id",
+            column_to_count=strata_variable,
             barcode_ref_column="TRANCHE_BARCODE_REF",
             tranche_column="tranche",
             group_by_columns=["cis_area_code_20", "enrolment_date"],
```

Comment:
The count will always be of unique households (ons_household_id), but the strata variables are those being grouped by here